libraries/Microsoft.Bot.Connector/Authentication/EmulatorValidation.cs

```diff
@@ -105,12 +107,16 @@ public static bool IsTokenFromEmulator(string authHeader)
         /// <remarks>
         /// A token issued by the Bot Framework will FAIL this check. Only Emulator tokens will pass.
         /// </remarks>
-        public static async Task<ClaimsIdentity> AuthenticateEmulatorToken(string authHeader, ICredentialProvider credentials, HttpClient httpClient, string channelId)
+        public static async Task<ClaimsIdentity> AuthenticateEmulatorToken(string authHeader, ICredentialProvider credentials, IChannelProvider channelProvider, HttpClient httpClient, string channelId)
         {
```

Comment:
Please update the comments above with the new parameter. 

Note: Generally I would reject this as this is a breaking change to a public method on a public class. In this case I cannot imagine anyone other than us calling this, so... enh.

Reply:
Done.